src/helm/benchmark/metrics/machine_translation_metrics.py

```diff
@@ -34,3 +35,50 @@ def evaluate_instances(self, request_states: List[RequestState]) -> List[Stat]:
             refs[0].append(request_state.instance.references[0].output.text)
         bleu_score = bleu.corpus_score(sys, refs).score
         return [Stat(MetricName("bleu")).add(bleu_score)]
+
+
+class CLEVAMachineTranslationMetric(Metric):
+    """
+    Compute the BLEU score for Machine Translation scenarios of CLEVA benchmark.
+    Based on sacrebleu, this implementation distinguishes target language and allows variable number of references.
+    If there are more than one hypothesis, only the first one is adopted in the calculation.
+    """
+
+    def evaluate_instances(self, request_states: List[RequestState]) -> List[Stat]:
+        """
+        Compute the corpus-level metric based on all reqeust_states.
+        """
+
+        def detect_language(request_states: List[RequestState]) -> str:
+            """
+            Determine the target language by detecting the language of references.
+            Currently, it only distinguishes if the target language is Chinese.
+            """
+
+            corpus: str = "".join(
+                [request_state.instance.references[0].output.text for request_state in request_states[:10]]
+            )
+            if detect(corpus) in ["zh-cn", "zh-tw"]:
+                return "zh"
+            else:
+                return "13a"  # Default tokenizer for sacrebleu.BLEU
+
+        bleu = BLEU(tokenize=detect_language(request_states))
+
+        max_num_references: int = max([len(request_state.instance.references) for request_state in request_states])
+        refs: List[List[str]] = [
+            [
+                request_state.instance.references[i].output.text if i < len(request_state.instance.references) else ""
+                for request_state in request_states
+            ]
+            for i in range(max_num_references)
+        ]
+
+        sys: List = []
```

Comment:
I think "sys" is not a good name because there is a frequently used Python package also called "sys". Try something like "hypos" etc.

Reply:
I choose "sys" because the original MachineTranslationMetric uses "sys." I think it may be better if we could follow their customs, but I do not have a strong opinion on this issue.

Reply:
OK then.